cmd/minishift/cmd/config/config.go

```diff
@@ -52,18 +52,21 @@ var settingsList []Setting
 
 var (
 	// minishift
-	ISOUrl           = createConfigSetting("iso-url", SetString, []setFn{validations.IsValidISOUrl}, []setFn{RequiresRestartMsg}, true, nil)
-	CPUs             = createConfigSetting("cpus", SetInt, []setFn{validations.IsPositive}, []setFn{RequiresRestartMsg}, true, nil)
-	Memory           = createConfigSetting("memory", SetString, []setFn{validations.IsValidMemorySize}, []setFn{RequiresRestartMsg}, true, nil)
-	DiskSize         = createConfigSetting("disk-size", SetString, []setFn{validations.IsValidDiskSize}, []setFn{RequiresRestartMsg}, true, nil)
-	VmDriver         = createConfigSetting("vm-driver", SetString, []setFn{validations.IsValidDriver}, []setFn{RequiresRestartMsg}, true, nil)
-	OpenshiftVersion = createConfigSetting("openshift-version", SetString, nil, nil, true, nil)
-	HostOnlyCIDR     = createConfigSetting("host-only-cidr", SetString, []setFn{validations.IsValidCIDR}, nil, true, nil)
-	DockerEnv        = createConfigSetting("docker-env", SetSlice, nil, nil, true, nil)
-	DockerEngineOpt  = createConfigSetting("docker-opt", SetSlice, nil, nil, true, nil)
-	InsecureRegistry = createConfigSetting("insecure-registry", SetSlice, nil, nil, true, nil)
-	RegistryMirror   = createConfigSetting("registry-mirror", SetSlice, nil, nil, true, nil)
-	AddonEnv         = createConfigSetting("addon-env", SetSlice, nil, nil, true, nil)
+	ISOUrl                = createConfigSetting("iso-url", SetString, []setFn{validations.IsValidISOUrl}, []setFn{RequiresRestartMsg}, true, nil)
+	CPUs                  = createConfigSetting("cpus", SetInt, []setFn{validations.IsPositive}, []setFn{RequiresRestartMsg}, true, nil)
+	Memory                = createConfigSetting("memory", SetString, []setFn{validations.IsValidMemorySize}, []setFn{RequiresRestartMsg}, true, nil)
+	DiskSize              = createConfigSetting("disk-size", SetString, []setFn{validations.IsValidDiskSize}, []setFn{RequiresRestartMsg}, true, nil)
+	VmDriver              = createConfigSetting("vm-driver", SetString, []setFn{validations.IsValidDriver}, []setFn{RequiresRestartMsg}, true, nil)
+	OpenshiftVersion      = createConfigSetting("openshift-version", SetString, nil, nil, true, nil)
+	HostOnlyCIDR          = createConfigSetting("host-only-cidr", SetString, []setFn{validations.IsValidCIDR}, nil, true, nil)
+	DockerEnv             = createConfigSetting("docker-env", SetSlice, nil, nil, true, nil)
+	DockerEngineOpt       = createConfigSetting("docker-opt", SetSlice, nil, nil, true, nil)
+	InsecureRegistry      = createConfigSetting("insecure-registry", SetSlice, nil, nil, true, nil)
+	RegistryMirror        = createConfigSetting("registry-mirror", SetSlice, nil, nil, true, nil)
+	AddonEnv              = createConfigSetting("addon-env", SetSlice, nil, nil, true, nil)
+	RemoteIPAddress       = createConfigSetting("remote-ip-address", SetString, nil, nil, true, nil)
```

Comment:
It should be just `--ip` . The usage will help user understand why `IP` is required.

Reply:
--ip conflicts in the way it gets interepreted, as we also have `network-ipaddress` for the static assignment for HyperV. please, for consistency we need to use `ipaddress` in the first place, but also group it logically; such as `--target-ipaddress` or `--deploy-ipaddress`, etc.

Reply:
I see you also use `provision` in the desctiptions. should this be the prefix?

Reply:
@gbraad we agreed on `remote-ip-address`